frontend/micro-ui/web/micro-ui-internals/packages/modules/im/src/pages/employee/ComplaintDetails.js

```
@@ -610,72 +671,102 @@ export const ComplaintDetails = (props) => {
+              <div className="TLComments">
+                <h3>{t("WF_SENDBACK_REASON")}</h3>
+                <h1>
+                  {arrNew[index]?.sendBackReason}
+                  {arrNew[index]?.sendBackSubReason ? ` - ${arrNew[index]?.sendBackSubReason}` : ""}
+                </h1>
+              </div>
+            ) : null}
+            {checkpoint?.wfComment ? (
+              <div>
+                {checkpoint?.wfComment?.map((e) => (
+                  <div className="TLComments">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)